File: 01-TextView/app/src/main/java/com/example/textapplication/MainActivity.java
package com.example.textapplication;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.util.Log;
import android.view.View;

public class MainActivity extends AppCompatActivity {
    private static final String TAG = "MainActivity";
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }
    
    public void showTextView(View v){
        Log.d(TAG, "showTextView: ====");
    }

    public void showButtonView(View view) {
        Intent it = new Intent(MainActivity.this,ButtonActivity.class);
        startActivity(it);

    }
}

File: 02-BroadcastDemo/app/src/main/java/alexiuce/github/com/broadcastapplication/MainActivity.kt
package alexiuce.github.com.broadcastapplication

import alexiuce.github.com.testlibrary.Person
import alexiuce.github.com.twolibrary.Two
import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import kotlinx.android.synthetic.main.activity_main.*


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        // 点击按钮,发送无序广播
        button.setOnClickListener{
            val intent = Intent()
            intent.action = "MYBroadcast"
            intent.putExtra("job","dev")
            sendBroadcast(intent)

            sendbr
        }

    }

    private fun example(){
        val p = Person("alex")
        println(p)
        val t = Two()
        t.say()
        }
}
